README.md

```diff
@@ -1 +1,143 @@
-# budget-app
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
```

Comment:
- [ ] Most of these links in your Table of Contents are broken. Kindly fix them by adding the sections or removing some that are irrelevant. But we should not have broken links on the README.

Reply:
Thanks for your suggestions 👍 I will fix it before the next review.